catalog/dags/data_refresh/reporting.py

```diff
@@ -30,9 +30,29 @@ def report_record_difference(before: str, after: str, media_type: str, dag_id: s
 Data refresh for {media_type} complete! :tada:
 _Note: All values are row estimates and are not (but nearly) exact_
 *Record count difference for `{media_type}`*: {before:,} → {after:,}
-*Change*: {count_diff:+,} ({percent_diff:+}% Δ)
+*Change*: {count_diff:+,} ({_round_sig_figs(percent_diff, 3):+}% Δ)
```

Comment:
As mentioned below, the `_round_sig_figs` can be removed if we simplify things using just format spec significant digits instructions.

If we use `g`, the general format, this will use scientific notation in some cases:

```
>>> "{n:+3g}".format(n=0.0000512346888129371512345)
'+5.12347e-05'
```

So, I think it would be better to remove the `* 100` on line 24, and use the `%` spec instead:

```
>>> "{n:+3%}".format(n=0.000000512346888129371512345)
'+0.000051%'
```

```suggestion
*Change*: {count_diff:+,} (percent_diff:+3%} Δ)
```

If you apply that suggestion, please update line 24 as well.

Reply:
FWIW the `3` here does not seem to have any effect 🤔 
```python
[nav] In [3]: "{n:+3%}".format(n=0.001000512346888129371512345)
Out[3]: '+0.100051%'

[nav] In [4]: "{n:+3%}".format(n=0.001000512340888129371512345)
Out[4]: '+0.100051%'

[nav] In [5]: "{n:+3%}".format(n=0.0000512340888129371512345)
Out[5]: '+0.005123%'

[nav] In [6]: "{n:+2%}".format(n=0.0000512340888129371512345)
Out[6]: '+0.005123%'

[nav] In [7]: "{n:+1%}".format(n=0.0000512340888129371512345)
Out[7]: '+0.005123%'

[nav] In [8]: "{n:+%}".format(n=0.0000512340888129371512345)
Out[8]: '+0.005123%'

```

Reply:
If `"{n:+3%}".format(num)` is the agreed way forward, I'm happy to make this change and update the tests.

Reply:
The number is apparently ignored for `%`, TIL. It does have an effect on general formatting, and I converted to `%` from that, not realising it wouldn't make a difference.

```
>>> "{n:+3g}".format(n=52.212331223456234512)
'+52.2123'
>>> "{n:+3g}".format(n=0.001000512346888129371512345)
'+0.00100051'
>>> "{n:+3g}".format(n=0.6888129371512345)
'+0.688813'
>>> "{n:+3g}".format(n=0.000000000000000000000000888129371512345)
'+8.88129e-25'
```

Reply:
Sara's right, I think this approach will be way more straightforward than anything I'm suggesting. If you're comfortable with making those changes Matt, please proceed!